mongoengine/connection.py

```diff
@@ -107,7 +111,6 @@ def get_connection(alias=DEFAULT_CONNECTION_NAME, reconnect=False):
             # Discard replicaSet if not base string
             if not isinstance(conn_settings['replicaSet'], basestring):
                 conn_settings.pop('replicaSet', None)
-            connection_class = MongoReplicaSetClient
```

Comment:
Does this mean that old versions of pymongo can no longer use the `MongoReplicaSetClient`?  That seems wrong and breaking for users of replicaSets.